packages/ibm-products/src/components/CoachmarkStack/CoachmarkStackHome.tsx

```diff
@@ -62,6 +139,24 @@ export let CoachmarkStackHome = forwardRef(
       );
     }
 
+    function renderNavLink(index, label, ref) {
```

Comment:
In the previous code ref is optional `ref = null`, can you change it like that?

Reply:
it was already set to `ref ?? null` a few lines below but I've reverted the change